lighthouse-core/test/audits/dobetterweb/no-old-flexbox-test.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const NoOldFlexboxAudit = require('../../../audits/dobetterweb/no-old-flexbox.js');
+const assert = require('assert');
+const stylesData = require('../../fixtures/styles-gatherer.json');
+
+/* eslint-env mocha */
+
+describe('Page does not use old CSS flexbox', () => {
+  it('fails when no input present', () => {
```

Comment:
maybe `it returns error value when no input present`? or something to differentiate "fail" due to an error vs "fail" due to having old flexbox